soh/soh/Enhancements/audio/AudioEditor.cpp

```diff
@@ -205,7 +204,7 @@ void Draw_SfxTab(const std::string& tabId, SeqType type) {
 
                 if (ImGui::Selectable(seqData.label.c_str())) {
                     CVarSetInteger(cvarKey.c_str(), value);
-                    LUS::RequestCvarSaveOnNextTick();
+                    LUS::Context::GetInstance()->GetWindow()->GetGui()->SaveConsoleVariablesOnNextTick();
```

Comment:
This is a bit verbose for how often it’s used, perhaps a macro?

Reply:
maybe? idk. at this point I don't want to think of a name/make sure we have the macro defined everywhere we use it (i'd expect it to be an SoH side thing, I don't think @Kenix3 would want to have that macro in LUS)

would you be cool with just making a code cleanup issue and having that be a post-merging-this-pr thing?

Reply:
Yeah that’s good

Reply:
https://github.com/HarbourMasters/Shipwright/issues/2943